### DISCUSSION TITLE
Meta_Hours_12

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#3644] [Meta-Hours]
![Screenshot 2022-08-02 at 11 11 05](https://user-images.githubusercontent.com/5887929/182338361-d99adb53-97d4-4f70-8b78-841b3aa3883b.png)

### Basilisk update
![image](https://user-images.githubusercontent.com/5887929/182410688-4923b99f-1c2f-47c2-90a6-cfed86f158f8.png)
📸 Check the amount of active wallets at [beta.kodadot.xyz/bsx/stats](https://beta.kodadot.xyz/bsx/stats/)

We are thank full for all Basilisk participants for our first phase of Testing.
It seems we got up to 89 active wallets. For the subsequent testing on Rococo, we will proceed with the same process, where the contributor and community can help us report bugs we internally might not encounter. **Valuable bug reports will be rewarded, of course.**
Gradually we will be heading to the last phase of testing on Rococo, where NFTs will be available for buying for KSM.  We hope for less complication, it will require updating few components, so we expect few dragons and snakes along the road as always.
- #3376 

### Revamp of KodaDot - heading to neu brutalism
![image](https://user-images.githubusercontent.com/5887929/182338706-49513018-bfe1-45c8-9585-be0b624c9669.png)
- We are getting onboard a young and potential designer we've met in Prague, Exez.  
- We won't leak much of the new design to the public until it is done in particular components which make sense, not tease you with prototypes which won't make it to production.
- We will be looking to start with revamping the landing site, which we feel is quite important, and we have been adding components a bit random due to our wild development. Then we will proceed to make gallery item detail which will become quite important component here. 
- We will have a form where you could put your feedback on what you want to see on KodaDot on the landing page and gallery item detail.

### Winning hackathon Polkadot North America with KodaDot: Moonsama 
![image](https://user-images.githubusercontent.com/5887929/182339879-355096a5-cfab-4cd7-9571-f51eb75d168f.png)
[tweet link](https://twitter.com/helloitsdamsky/status/1553319594496073728)
- **We are thrilled that we've won first prize for integrating Moonsama**, an EVM-based collection on MoonRiver, and it will help us to open gates to other integrations among other EVM-based parachains, namely **MoonRiver**, integrating them one by one and continuing to the MoonBeam and Astar EVM. 
- **We are actively looking for EVM-based collections.** You can reach out to @JustLuuuu, and she can help you to arrange a campaign with KodaDot once our EVM support is there, and you can secure spotlighted area among the first listed EVM collections. 
- We will start gradually revealing a read-only functionality and then adding actions so you can log in with your EVM wallet (Talisman, Metamask, SubWallet &c)
- We will be doing a campaign for incentivized testing, same as for Basilisk, once things are in place and working :)
- You can keep track of integrating MoonRiver & MoonBeam below, it will be live issue and we will be editing that one
- https://github.com/kodadot/nft-gallery/issues/3647 

### Where is KodaDot going to next?
- Thanks to Polkadot, there are abundant opportunities where to go and what to integrate. 
- We are looking way to **integrate [Acala EVM+](https://evmdocs.acala.network/general/about-acala-evm+)**, which brings on table new features like pay bring your own gas" (paying fees in any tokens like a stablecoin), and there’s an on-chain automatic scheduler that enables use cases like subscription and recurring payments etc.
We are exploring **Astar/Shiden integration**, to bring WASM contracts to the KodaDot. It's not overnight integration, and it might take a while. Till then, we've seen there is the possibility of integrating the RMRK runtime pallet, which would open excellent opportunities in the Dotsama ecosystem for other NFT Dapps.

### Generative Art Campaign
We've got access to Dall-e 2, so we are actively exploring ways to mint premium assembled collections. We would like to get **prompt artists** on board to help with suggestions and what to mint ofc. 
@petersopko is actively playing with prompts within Dall-e 2, so I bet he can expand on that about it below. 
 
### Bugs and reports
As KodaDot is getting a complex app with a lot of lines of code and a lot of contributors, we thrive on gathering user feedback and reporting bugs you may encounter during using KodaDot.
We value recent contributors who made an effort to report bugs and raise awareness. We are always happy to reward valuable reports on bugs which we aren't aware of or it's still happening on our beta version. By keeping this up, you can help make KodaDot a better experience for yourself and others.
 
- [40 open reports](https://github.com/kodadot/nft-gallery/issues?q=is%3Aopen+is%3Aissue+label%3Abug) 
- [380 resolved bug reports](https://github.com/kodadot/nft-gallery/issues?q=is%3Aissue+label%3Abug+is%3Aclosed)

### Looking for Polkadot & Kusama community in Slovenia
![image](https://user-images.githubusercontent.com/5887929/182343173-7729efb6-e683-48bc-9bb3-2e4bc72d93fa.png)
![telegram-cloud-photo-size-4-6001520251705081052-y](https://user-images.githubusercontent.com/5887929/182346870-30f4f0b5-d24d-450c-8487-f9cb66d95990.jpg)

We are gradually preparing web3 cowork in Bled, Slovenia, which will be focused on influential Substrate builders in the ecosystem. We've picked this place as we love the air, climate, nature and many opportunities for our wellbeing; it's not your seasonal destination but a year-round place to be; come along with your team to have power sessions and brainstorm new strategies or products for your Polkadot products. We will release more information once things are completed, so stay tuned!
This post was [made from SubWork](https://subwork.xyz) in Slovenia.

**Comment by petersopko** [comment]
Peter's Meta Minutes:

- automated tests
- updates in repo
- first DALL•E 2 collection minted
- https://beta.kodadot.xyz/rmrk/collection/A4598F5615F201D739-UDF
- prompts and the future of prompting
- http://dallery.gallery

in the meantime:

![DALL·E 2022-08-02 12 10 07 - Digital art of portrait of three chimpanzees wearing VR headsets, having fun, excited facial expression, head-and-shoulders shot, white background, cu](https://user-images.githubusercontent.com/73316633/182351081-1985b38e-d901-4add-ae5a-d07e719d3867.png)

**Comment by DeluneClaire** [comment]
Claire's minute:
- Gleam campaign
- Surprise 1 (CWH)
- Surprise 2 (Yumi)
- Surprise 3 (TikTok)
- Maybe surprise 4? (T-shirt design)

**Comment by JustLuuuu** [comment]
Luuu:
- Reddit, how is it going (next event)
- new collabs with artist
- Subsquid collab
- NFTs - t-shirts with our old design

**Comment by vikiival** [comment]
**Viki's Meta Minutes**
-

### DEV in numbers:
`is:pr is:merged created:>=2022-06-20`

> 149 PRs merged

### Internal:

We had a Hands-on dev call #3615, and we went through the good coding practices as some parts of the code are really hard to read.

Developers that joined the hands-on call are writing better code than previously.
This helps us refactor the code mentioned #3311 towards the new Vue3/Nuxt3 and the NeoBrutalism style that @yangwao has described.

### Refactor

It may not seem urgent, but we are in a state where we need that. Why? 

**From the project perspective**

- It is easier to implement other parachains


**From the developer perspective**

- We will have less code that needs to be maintained.
- Code will be easier to read
- Can onboard new devs more seamlessly

**From the user perspective**

- faster application
- fewer bugs.
- more performance

<img width="544" alt="Screenshot 2022-08-03 at 17 03 16" src="https://user-images.githubusercontent.com/22471030/182656199-77530458-2702-47d8-b6d9-54371b4c70e3.png">

### FireSquid:

![tenor](https://user-images.githubusercontent.com/22471030/182624690-7c0e73e9-61bf-44f7-b57d-0b7eb16abf8a.png)

We went for a 20-min adventure to migrate our existing SubSquid RMRK indexer, Rubick, to the FireSquid. This should bring faster processing times and the possibility of supporting RMRKv2. 

Although It seemed like an easy task in the first place, it was kind of a heavy lift.

<img width="251" alt="Screenshot 2022-08-03 at 14 42 13" src="https://user-images.githubusercontent.com/22471030/182624993-9f0679df-0dc1-4062-8598-b47ec970b377.png">

We also used this migration to learn how the new FireSquid works so we can bring new optimised indexers for BSX, Moonriver and Astar.

### Sunsetting Magick

Since May 2021, we have used a SubQuery indexer for RMRK NFRs called magick. As this was the stable backbone for our NFT gallery for many months. We couldn't use many cool features we already use on SubSquid, and the upgrades for the magick were a bit tricky.

The third reason we're the incompatible queries across the application added a lot of noise to the codebases.

Therefore we are using the Rubick as the main indexer (#2929).

**Comment by damskyftw** [comment]
Damsky time

- Where we headed
- Presenting [Community viki](https://www.notion.so/metaprime/KodaDot-Guilds-Community-overview-e8eb64f1bbd74e7a924c5fb07e3ff3dd)
- Launching **Writer guild**
- What are guilds anyway
- Changes on discord
- Junior Ambassadors
- Community session next week - open invite
